lib/solarwinds_apm/otel_config.rb

```diff
@@ -40,7 +40,7 @@ def self.resolve_sampler
 
     def self.resolve_sampler_config
       sampler_config = {}
-      sampler_config["trigger_trace"] = "enabled"
+      sampler_config["trigger_trace"] = SolarWindsAPM::Config[:trigger_tracing_mode]
       sampler_config["trigger_trace"] = nil if ENV["SW_APM_TRIGGER_TRACING_MODE"] == 'disabled'
```

Comment:
Make sense that an env var disabled would override the config file settings, but now I'm wondering why `disabled` would make it `nil` instead of `:disabled`.

Reply:
Yes, `:disabled` is more readable (changed)